config/config.go

```diff
@@ -875,7 +875,7 @@ func (cfg *ConsensusConfig) ValidateBasic() error {
 // TxIndexConfig
 
 // TxIndexConfig defines the configuration for the transaction indexer,
-// including tags to index.
+// including compositeKeys to index.
```

Comment:
I think we'll need to explain what compositeKey is here:

```
Remember that Event have a following structure:
type: [
  key: value,
  ...
]
```

composite key is a type and key, separated by a single dot. For example, given the Event below

```
"jack": [
  "account.number": 100
]
``` 

composite key will be `jack.account.number`